arrows/ffmpeg/ffmpeg_video_input.cxx

```diff
@@ -753,53 +816,54 @@ class ffmpeg_video_input::priv
       int64_t frame_ts = this->f_video_stream->duration + this->f_start_time;
       do
       {
-        auto seek_rslt = av_seek_frame(this->f_format_context,
-                                       this->f_video_index,
-                                       frame_ts,
-                                       AVSEEK_FLAG_BACKWARD);
-        avcodec_flush_buffers(this->f_video_encoding);
+        auto seek_rslt = av_seek_frame( this->f_format_context,
+                                        this->f_video_index,
+                                        frame_ts,
+                                        AVSEEK_FLAG_BACKWARD );
+        avcodec_flush_buffers( this->f_video_encoding );
 
-        if (seek_rslt < 0)
+        if( seek_rslt < 0 )
         {
           break;
         }
 
         advance_successful = this->advance();
 
-        // Continue to make seek request further back until we land at a valid frame
+        // Continue to make seek request further back until we land at a valid
+        // frame
         frame_ts -= this->f_backstep_size * this->stream_time_base_to_frame();
-      } while (!advance_successful);
+      } while( !advance_successful );
 
-      LOG_DEBUG(this->logger, "Seeked to near end frame: " << this->frame_number());
+      LOG_DEBUG( this->logger,
+                 "Seeked to near end frame: " << this->frame_number() );
 
       // Step through the end of the video to find the last valid frame number
       do
       {
         number_of_frames = this->frame_number();
-      }
-      while (this->advance());
+      } while( this->advance() );
```

Comment:
BTW, are we okay with this? I believe Uncrustify can be configured to break here instead, though I think not breaking is clearer (and it seems even this file previously used both styles).

My vote is for "no break" (i.e. keep the changes as applied), but just thought I'd point it out.